pkg/steps/template.go

```diff
@@ -442,30 +442,27 @@ func createOrRestartPod(podClient ctrlruntimeclient.Client, pod *coreapi.Pod) (*
 	return pod, nil
 }
 
-func waitForPodDeletion(podClient ctrlruntimeclient.Client, namespace, name string, uid types.UID) error {
-	timeout := 600
-	for i := 0; i < timeout; i += 2 {
+func waitForPodDeletion(ctx context.Context, podClient ctrlruntimeclient.Client, namespace, name string, uid types.UID) error {
+	return wait.ExponentialBackoffWithContext(ctx, wait.Backoff{Duration: 10 * time.Second, Factor: 2, Steps: 10}, func() (done bool, err error) {
```

Comment:
10 times duplicating the duration, starting with 10 gives a total of about 10k seconds intead of the 600 seconds we had before or am I misunderstanding this?
/hold

Reply:
Oops, I wanted 1024, sorry -.-